### PR TITLE
Acquire and parse US cities data

### PR DESCRIPTION
**Pull request by tangaw**
(*New PR after rebasing master)
Addresses #25 
- `src/data/ref/uscities.csv` -- raw data
- `src/data/parseCities.ipynb` -- notebook used to do basic parsing of raw data
- `src/data/ref/parsed_us_cities.csv` -- parsed data; columns included:
```
'city_ascii', 'state_id', 'county_fips', 'county_name', 
'lat', 'lng', 'population', 'density', 
'military', 'timezone', 'zips', 'id'
```

### CODE REVIEW

**Comment by tangaw** [comment]
Notes added within an `EXPLANATION.md` file to explain a couple variables including the two you mentioned.
We can certainly just use the `uscities.csv` file. I've removed the redundancies.

Ready for next round of review.

File: src/data/EXPLANATION.md [hunk 1]
```diff
@@ -0,0 +1,11 @@
...
+This is the free/basic version of the dataset, and attribution to the original website will be required.
+
+### Ambiguous variable explanations
+- **city** - City name
+- **city_ascii** - City name converted to ascii (*recommended*)
+- **country_fips** - The 5-digit FIPS code for the primary county. The first two digits correspond to the state's FIPS code.
+- **Density** - The estimated population per square kilometer
+- **Military** - `TRUE` if this place is a military establishment such as a fort or base.
+- **zips** - A string containing all five-digit zip codes in the city/town, delimited by a space.
+- **Id** - Unique ID for each city
```
Comment (dongskyler) [inline src/data/EXPLANATION.md:11]:
```suggestion
- **Id** - Unique ID for each city

```

Comment (dongskyler) [reply src/data/EXPLANATION.md:11]: Leave a blank line at the end of the code

File: src/data/EXPLANATION.md [hunk 2]
```diff
@@ -0,0 +1,11 @@
+### [Source](https://simplemaps.com/data/us-cities)
```
Comment (dongskyler) [inline src/data/EXPLANATION.md:1]:
```suggestion
# EXPLANATION

## [Source](https://simplemaps.com/data/us-cities)

```

File: src/data/EXPLANATION.md [hunk 3]
```diff
@@ -0,0 +1,11 @@
...
+This is the free/basic version of the dataset, and attribution to the original website will be required.
+
+### Ambiguous variable explanations
```
Comment (dongskyler) [inline src/data/EXPLANATION.md:4]:
```suggestion
## Ambiguous variable explanations

```

Comment (dongskyler) [reply src/data/EXPLANATION.md:4]: Syntax: Headings need to be surrounded by blank lines

File: src/data/EXPLANATION.md (hunk 2 above)
Comment (dongskyler) [reply src/data/EXPLANATION.md:1]: The first line of a markdown file needs to be a top level heading.

Headings need to be surrounded by blank lines.

You can install a markdown linting extension in VS Code, such as: https://marketplace.visualstudio.com/items?itemName=DavidAnson.vscode-markdownlint

**Pull request merged**